internal/coremsgs/en_config_descriptions.go

```diff
@@ -20,301 +20,265 @@ import (
+	ConfigDatabasePostgresMaxIdleConns    = ffc("config.database.postgres.maxIdleConns", "The maximum number of idle connections to the database", intType)
+	ConfigDatabasePostgresURL             = ffc("config.database.postgres.url", "The PostgreSQL connection string for the database", stringType)
+
+	ConfigDatabaseSqlite3MaxConnIdleTime = ffc("config.database.sqlite3.maxConnIdleTime", "The maximum amount of time a database connection can be idle", timeDurationType)
+	ConfigDatabaseSqlite3MaxConnLifetime = ffc("config.database.sqlite3.maxConnLifetime", "The maximum amount of time to keep a database connection open", timeDurationType)
+	ConfigDatabaseSqlite3MaxConns        = ffc("config.database.sqlite3.maxConns", "Maximum connections to the database", intType)
+	ConfigDatabaseSqlite3MaxIdleConns    = ffc("config.database.sqlite3.maxIdleConns", "The maximum number of idle connections to the database", intType)
+	ConfigDatabaseSqlite3URL             = ffc("config.database.sqlite3.url", "The SQLite connection string for the database", stringType)
+
+	ConfigDataexchangeType = ffc("config.dataexchange.type", "The Data Exchange plugin to use", stringType)
+
+	ConfigDataexchangeFfdxInitEnabled     = ffc("config.dataexchange.ffdx.initEnabled", "Instructs FireFly to always post all current nodes to the `/init` API before connecting or reconnecting to the connector", booleanType)
+	ConfigDataexchangeFfdxManifestEnabled = ffc("config.dataexchange.ffdx.manifestEnabled", "Determines whether to require+validate a manifest from other DX instances in the network. Must be supported by the connector", stringType)
+	ConfigDataexchangeFfdxURL             = ffc("config.dataexchange.ffdx.url", "The URL of the Data Exchange instance", "URL "+stringType)
+
+	ConfigDataexchangeFfdxProxyURL = ffc("config.dataexchange.ffdx.proxy.url", "The URL of the Data Exchange HTTP proxy", "URL "+stringType)
+
+	ConfigDebugPort = ffc("config.debug.port", "An HTTP port on which to enable the go debugger", intType)
+
+	ConfigDownloadWorkerCount       = ffc("config.download.worker.count", "The number of download workers", intType)
+	ConfigDownloadWorkerQueueLength = ffc("config.download.worker.queueLength", "The length of the work queue in the channel to the workers - defaults to 2x the worker count", intType)
+
+	ConfigEventAggregatorBatchSize            = ffc("config.event.aggregator.batchSize", "The maximum number of records to read from the DB before performing an aggregation run", byteSizeType)
+	ConfigEventAggregatorBatchTimeout         = ffc("config.event.aggregator.batchTimeout", "How long to wait for new events to arrive before performing aggregation on a page of events", timeDurationType)
+	ConfigEventAggregatorFirstEvent           = ffc("config.event.aggregator.firstEvent", "The first event the aggregator should process, if no previous offest is stored in the DB. Valid options are `oldest` or `newest`", stringType)
```

Comment:
Think this is defunct config now - the key can be removed (not queried anywhere).

Reply:
This one is still queried here https://github.com/hyperledger/firefly/blob/main/internal/events/aggregator.go#L81

Reply:
Sorry, my bad -agreed looking at the code. I don't thin it makes sense to keep it